user_new.inc.php

```diff
@@ -113,8 +113,8 @@ function GetLicenceComboBox($lic_name, $lic_value)
 else
 {
 	$value = '<select name=\'poh\'>';
-	$value .= '<option value=\'H\''.(($zaznam["poh"]==GC_KATEG_M) ? ' SELECTED' : '').'>'.GC_KATEG_M.'</option>';
-	$value .= '<option value=\'D\''.(($zaznam["poh"]==GC_KATEG_W) ? ' SELECTED' : '').'>'.GC_KATEG_W.'</option>';
+	$value .= '<option value=\'H\' '.(($zaznam["poh"]=="H") ? 'SELECTED' : '').'>'.GC_KATEG_M.'</option>';
```

Comment:
Tady se nabizi otazka, zda se v db ma ukladat D a H, nebo to co je v danem oddile zvoleno za D a H.
Sam si ted z hlavy nejsem jist